mantaray/persist_test.go

```diff
@@ -0,0 +1,93 @@
+package mantaray_test
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"sync"
+	"testing"
+
+	"github.com/ethersphere/manifests/mantaray"
```

Comment:
Wrong import, `manifests` -> `manifest`.

Reply:
hmm how did this work?